main.tf

```diff
@@ -38,20 +38,26 @@ data "aws_iam_policy_document" "default" {
 }
 
 module "s3_bucket" {
-  source                 = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.7.0"
-  enabled                = var.enabled
-  namespace              = var.namespace
-  stage                  = var.stage
-  environment            = var.environment
-  name                   = var.name
-  region                 = var.region
-  acl                    = var.acl
-  policy                 = join("", data.aws_iam_policy_document.default.*.json)
-  force_destroy          = var.force_destroy
-  versioning_enabled     = true
-  lifecycle_rule_enabled = false
-  delimiter              = var.delimiter
-  attributes             = var.attributes
-  tags                   = var.tags
-  lifecycle_prefix       = var.lifecycle_prefix
+  source                             = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.7.0"
+  enabled                            = var.enabled
+  namespace                          = var.namespace
+  stage                              = var.stage
+  environment                        = var.environment
+  name                               = var.name
+  region                             = var.region
+  acl                                = var.acl
+  policy                             = join("", data.aws_iam_policy_document.default.*.json)
+  force_destroy                      = var.force_destroy
+  versioning_enabled                 = true
+  lifecycle_rule_enabled             = var.lifecycle_rule_enabled
+  enable_glacier_transition          = var.enable_glacier_transition
+  expiration_days                    = var.expiration_days
+  glacier_transition_days            = var.glacier_transition_days
+  noncurrent_version_expiration_days = var.noncurrent_version_expiration_days
+  noncurrent_version_transition_days = var.noncurrent_version_transition_days
+  standard_transition_days           = var.standard_transition_days
```

Comment:
these options are not exposed in the terraform-aws-s3-log-storage so test is failing, so there is more work to do upstream, I will work on PR today

Reply:
I was wrong, they are exposed but the module version source needs to be updated @deadlysyn